pkg/constants/constants.go

```diff
@@ -506,16 +506,19 @@ const (
 	OpenTelemetryCollector  = "OpenTelemetryCollector"
 )
 
-// Model Parallel Options
+// MultiNode environment variables
 const (
 	TensorParallelSizeEnvName   = "TENSOR_PARALLEL_SIZE"
 	PipelineParallelSizeEnvName = "PIPELINE_PARALLEL_SIZE"
+	RayNodeCountEnvName         = "RAY_NODE_COUNT"
+	RequestGPUCountEnvName      = "REQUEST_GPU_COUNT"
 )
 
-// Model Parallel Options Default value
+// MultiNode default values
 const (
-	DefaultTensorParallelSize   = "1"
-	DefaultPipelineParallelSize = "2"
+	DefaultTensorParallelSize   = 1
+	DefaultPipelineParallelSize = 1
+	DefaultGPUCount             = 1
```

Comment:
What would be the difference between `DefaultTensorParallelSize| DefaultPipelineParallelSize ` and `DefaultGPUCount`?
if there are two GPUs in the cluster, tensor should not be bigger than 2, right?
What is the usage of defaultGPUCount?

Reply:
Where is it being used?
You added on this PR, but I didn't find it anywhere.